shared/teams/new-team/wizard/new-team-info.tsx

```diff
@@ -199,14 +200,17 @@ const NewTeamInfo = () => {
                     selectedRole={selectedRole}
```

Comment:
there's another arg, I think `presetRole` maybe, that you should pass the `realRole` to